styles/styles.css

```diff
@@ -335,17 +346,31 @@ a:hover {
   padding: 120px 7.22% 120px 7.22%;
 }
 
+.logo__footer {
+  width: 98px;
+  height: 19px;
+  color: white;
+}
+
+.footer .logo {
+  display: none;
+}
+
+.logo__link {
+  display: inherit;
+}
+
 .footer__menu {
   display: flex;
   flex-wrap: wrap;
-  column-gap: 1.11%;
-  row-gap: 3vh;
+  column-gap: 16px;
+  row-gap: 6vh;
```

Comment:
It's not recommended to use vh for line spacing because each monitor has different extensions. So in this case it's better to use px